Sources/NIOCore/AddressedEnvelope.swift

```diff
@@ -62,8 +62,10 @@ extension AddressedEnvelope: Equatable where DataType: Equatable {}
 
 extension AddressedEnvelope: Hashable where DataType: Hashable {}
 
+extension AddressedEnvelope: NIOSendable where DataType: NIOSendable {}
```

Comment:
This conformance isn't quite right: we don't need the `DataType` to conform to `NIOSendable`, we only need it to conform to `Sendable`. This will make this declaration more annoying to write, but we should still spell it correctly.

Reply:
Oh shoot so it is, good catch.